packages/client/utils/makeDefaultTeamName.ts

```diff
@@ -0,0 +1,54 @@
+export const DEFAULT_TEAM_NAMES = [
+  'Bug Writers 🪲',
+  'Long Meeting Lovers ❣️',
+  'Work Procrastinators ⏱️',
+  'Eat Lunch at 11AM 🥪',
+  'Midday Nap 😴',
+  'Show Us Your Cat 🐱',
+  'Comb Your Hair for Zoom 💅',
+  'Pajama Pants🌛',
+  'Highly Caffeinated ☕',
+  'Mute Slack & Chill 😎',
+  'Friday Afternoon Meetings Should Be Illegal 🙈',
+  "MacGuyver's of Fixing Bugs 🛠️",
```

Comment:
That should be MacGuyvers, shouldn't it? As it's just the plural

Reply:
Fixed! 🙏